### PR TITLE
fixing chatbot tests, chatbot makefile and common makefile

### PR DESCRIPTION
**Pull request by Gregory-Pereira**

### CODE REVIEW

**Comment by rhatdan** [comment]
Can you rebase your PR?

File: .github/workflows/chatbot.yaml [hunk 1]
```diff
@@ -58,6 +58,10 @@ jobs:
         working-directory: ./recipes/natural_language_processing/${{ env.IMAGE_NAME }}
         run: make install
 
+      - name: Download model
+        working-directory: ./model_servers/${{ matrix.image_name }}/
+        run: make download-model-mistral
```
Comment (rhatdan) [inline .github/workflows/chatbot.yaml]: I think this is blowing up in PR?

Comment (Gregory-Pereira) [reply .github/workflows/chatbot.yaml]: Yeah, but I also realized that it requires both chrome and the chrome driver with a paired version, Ive been reworking this.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**